src/mmrm_spatial.cpp

```diff
@@ -0,0 +1,134 @@
+#include "init.h"
```

Comment:
instead of starting a separate file, it would be better to only have if/else separation as needed for euclidean vs. ordered covariance structures within the old file/ program